lib/hu_board/services/slack.rb

```diff
@@ -26,10 +26,17 @@ def github_url
       ENV["GITHUB_WEB_ENDPOINT"] || "https://github.com/"
     end
 
+    def avatar_url(mash)
+      if github_url != 'https://github.com/'
```

Comment:
@rauhryan @dahlbyk this is the best way I know to differentiate between enterprise and not on the fly, if theres a better key let me know.